app/assets/javascripts/angular/directives/graph_chart.js

```diff
@@ -277,6 +278,19 @@ angular.module("Prometheus.directives").directive('graphChart', ["$location", "W
         rsGraph.series.legend = legend;
         rsGraph.render();
 
+        var drag = new Rickshaw.Graph.DragZoom({
+          graph: rsGraph,
+          opacity: 0.5,
+          fill: 'steelblue',
+          minimumSelection: 25,
```

Comment:
makes sense to me. I'll add the functionality and see what @juliusv thinks should be our minimum allowed zoom window